libs/sdk-ui-tests-e2e/cypress/tools/table.ts

```diff
@@ -208,4 +208,11 @@ export class Table {
         cy.get(".s-header-cell-label span").invoke("width").should("equal", width);
         return this;
     }
+
+    notUnderlined(item: string) {
```

Comment:
change name method to: `isCellUnderlined`, add one more argument `expect` to check in case underline-> that `have.css`